internal/cli/atlas/clusters/connectionstring/description.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package connectionstring
+
+const (
+	connectionString = "Manage MongoDB cluster connection string"
+	describe         = "Retrieve the connection string of your MongoDB cluster."
```

Comment:
let's make this clear 
```suggestion
	describe         = "Retrieve the SRV connection string of your MongoDB cluster."
```